CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## next version 
 - Revert "Adds limited model contract enforcement"
+- 
```

Comment:
Would you update the line with adding the title?

Reply:
Thanks for catching this, I updated the line.